bokeh/server/callbacks.py

```diff
@@ -158,44 +175,44 @@ def __init__(self, document, callback, timeout, id=None):
         self._timeout = timeout
 
     @property
-    def timeout(self):
+    def timeout(self) -> int:
         ''' The timeout (in milliseconds) that the callback should run after.
 
         '''
         return self._timeout
 
-    def _copy_with_changed_callback(self, new_callback):
+    def _copy_with_changed_callback(self, new_callback: Callback) -> TimeoutCallback:
         ''' Dev API used to wrap the callback with decorators. '''
         return TimeoutCallback(self._document, new_callback, self._timeout, self._id)
 
 #-----------------------------------------------------------------------------
 # Private API
 #-----------------------------------------------------------------------------
 
-class _DocumentCallbackGroup:
+class DocumentCallbackGroup:
```

Comment:
Needs to be moved above Private API comment line if so

Reply:
I changed this, because this is in fact not a private (to a module) API, but used elsewhere, whereas `_` is a clear indicator of a private API. Though I'm not sure if it's a dev API as well, perhaps an internal API. If we treat `_` as an internal API indicator instead, then I can revert this. Though various linters would warn about this.